### PR TITLE
added Irish flag

### PR DESCRIPTION
**Pull request by JamesKemp296**
I added the Irish flag by reverse engineering France, AKA just changing the colors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,6 +97,33 @@
...
+      }
+      .ireland-stripe:nth-child(2){
+        background-color:#FFFFFF;
+      }
+      .ireland-stripe:nth-child(3){
+        background-color:#E9863F;
+      }
+      #sweden{
+        height: 61vw;
+        grid-template-columns: 5fr 2fr 11.2fr;
```
Comment (andyweiss1982) [inline index.html]: Can we keep the frs at integer values?

File: index.html [hunk 2]
```diff
@@ -97,6 +97,33 @@
...
+      }
+      .swed-bl{
+        background-color: #3668A1;
+      }
+      .swed-yl{
+        background-color: #ECC505;
+      }
+      .stripe-full{
+        grid-column: 1 / -1;
+      }
```
Comment (andyweiss1982) [inline index.html]: Can we move this declaration to the top of the styling -- right under `.flag` -- since it's used in many different flags?

**Review by andyweiss1982 — COMMENTED** [review]
Looking pretty good, almost ready for merge.  Just a few small comments.

Also, can you add descriptions of the proportions to the README, so that it matches the other flags listed there?

**Review by andyweiss1982 — APPROVED** [review]

**Comment by aaronholla** [comment]
@JamesKemp296 Awesome job! 🇮🇪 🍀

**Pull request merged**